run.py

```diff
@@ -15,38 +16,135 @@
                     datefmt="%Y-%m-%d %I:%M:%S %p %Z")
 
 
-def setupVA(va_target):
+def parseCmdArgs():
```

Comment:
We're missing unit-tests for all of this logic.  I'd recommend making the  command parsing it's own class to condense this and make it testable.